.changeset/slow-swans-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@propeldata/ui-kit': minor
+---
+
+Add sql query hook
```

Comment:
Should there be two of these? Or just one?

Reply:
Somehow a conflict occured on the repo. 

So, I will fix it.